tests/storage/restricted_namespace_cloning/utils.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+import pytest
+from kubernetes.client.rest import ApiException
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.datavolume import DataVolume
+
+from tests.storage.restricted_namespace_cloning.constants import TARGET_DV
+from tests.storage.utils import assert_pvc_snapshot_clone_annotation
+from utilities.constants import PVC
+from utilities.storage import (
+    ErrorMsg,
+    create_dv,
+    is_snapshot_supported_by_sc,
+    sc_volume_binding_mode_is_wffc,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+
+def verify_snapshot_used_namespace_transfer(cdv: DataVolume, unprivileged_client: DynamicClient) -> None:
+    storage_class = cdv.storage_class
+    # Namespace transfer is not possible with WFFC
```

Comment:
Should this be skipped in this situation?

Reply:
This function is validating namespace transfer, why would you call it for WFFC then?

Reply:
What I am saying is, don't call it for wffc.

Reply:
Having separate functions for separate checks would be cleaner.

Reply:
Currently this function is being called in tests like test_create_vm_with_cloned_data_volume_permissions_for_pods_positive(), test_create_vm_with_cloned_data_volume_positive() and test_user_permissions_positive(), without any other calls. My concern is, if the tests are not doing anything because these conditions are not met, we are not catching it. But I think you need to address it with dedicated global configs or some other approach when you or your team handles removal of programmatic skips from T2/T3. It would be out of scope for this PR for the time being.